javapeople.yaml

```diff
@@ -885,6 +885,10 @@
   twitter: "thjanssen123"
   fediverse: "thjanssen123@fosstodon.org"
   github: "thjanssen"
+- name: "Tim Yates"
+  twitter: "tim_yates"
+  fediverse: "tim_yates@foojay.social
```

Comment:
I think a " is missing here, that's why the build fails.

Reply:
😿 

Thanks!  Heh, I managed to squeeze a bug into 4 lines!

And then said "3 lines" in the commit message...

I should go back to bed and call today off ;-)